android/rnkiwimobile/build.gradle

```diff
@@ -79,14 +76,19 @@ dependencies {
     androidTestImplementation "com.android.support.test.espresso:espresso-core:$espressoVersion"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
 
-    if (enableHermes) {
-      def hermesPath = "../../node_modules/hermesvm/android/";
-      debugImplementation files(hermesPath + "hermes-debug.aar")
-      stagingReleaseImplementation files(hermesPath + "hermes-release.aar")
-      releaseImplementation files(hermesPath + "hermes-release.aar")
-    } else {
-      implementation jscFlavor
-    }
+    implementation "com.kiwi.android:hermesvm:$hermesVmVersion-SNAPSHOT"
```

Comment:
This means we will always use the deployed version right?

Could I add this: 

```
def hermesPath = "../../node_modules/hermesvm/android/";	
      debugImplementation files(hermesPath + "hermes-debug.aar")	
      stagingReleaseImplementation files(hermesPath + "hermes-release.aar")
releaseImplementation "com.kiwi.android:hermesvm:$hermesVmVersion-SNAPSHOT"

```

Since we will also use this in the playground app.

Reply:
Sure, I thought the rnkiwimobile is used only for stagingRelease playground and kiwi.com app.

Reply:
There is this snipped in `/playground/build.gradle`:

```
debugImplementation(project(':rnkiwimobile')) {
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }

    // You can choose staging to use the Staging CodePush key
    stagingReleaseImplementation(project(':rnkiwimobile')) {
        exclude group: "com.android.support"
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }

    releaseImplementation("com.trinerdis.skypicker:rnkiwimobile:$rnKiwiMobileVersion-SNAPSHOT") {
        exclude group: "com.android.support"
        exclude group: "com.google.android.gms"
        exclude group: "com.google.firebase"
    }
```

which means it compiles rnkiwimobile right?